### DISCUSSION TITLE
Calendar view is taking too long to load in airflow 2.3.0

### DISCUSSION THREAD

**Discussion by SamanwayB** [apache/airflow#23545] [General]
### Apache Airflow version

2.3.0 (latest released)

### What happened

Calendar view in DAG is taking too long to load. It was fine in 2.2.5.

### Operating System

MacOS 12.3

### Deployment

Other 3rd-party Helm chart

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by eladkal** [comment]
Please add more information

**Comment by potiuk** [comment]
Agree with @eladkal . Converting it to a discussion. 

* What's too long? 
* How does it compare?
* What kind of setup, configuration and dags you have that trigger this behaviour?
* Any logs you can provide from this time when you have the problem (javascript console logs etc.).
* Any movies recording showing the difference in behaviour you can provide?

We are simply unable to act on "Calendar view in DAG is taking too long to load. It was fine in 2.2.5.".

**Reply by SamanwayB** [reply]
- I am using Kubernetes Executor for my airflow DAGs. 
 - I have a DAG which runs in every 15 minutes. When I open calendar view of the DAG it takes at least 3 minutes to load, it used to be swift. I did not change any configs in between, just upgraded the airflow version to 2.3.0. 
 - I have other DAGs with less frequent schedule(once a day), those are fine. Calendar view opens in no time for them.
 - I have a backend of postgresql 13.4.

**Reply by potiuk** [reply]
@bbovenzi -> might be interesting to see if this can be improved. Seems that there could be some latency of API calls that comes into play when there is a high frequency DAG.

**Reply by potiuk** [reply]
Created back an issue from the discussion https://github.com/apache/airflow/issues/23602
